utility/map_str_enhancer.go

```diff
@@ -41,6 +35,10 @@ func (enh MapStrEnhancer) Add(m common.MapStr, key string, val interface{}) {
 		if valArr := val.([]string); len(valArr) > 0 {
 			m[key] = valArr
 		}
+	case []common.MapStr:
```

Comment:
why this new case is added?

Reply:
To get rid of the `len(st) > 0` checks we had for adding the `stacktrace` to `transactions` and `errors`.